roles/generate-docs/templates/docs-template.j2

```diff
@@ -0,0 +1,26 @@
+# Labs Residency Environment
```

Comment:
I'd suggest adding this template as part of a sample inventory and use it from there.